source/_patterns/01-elements/headline/headline.scss

```diff
@@ -18,27 +12,64 @@ h1,
 h2,
 h3 {
 	&.elm-headline {
-		font-weight: 700;
-	}
-}
-h4 {
-	&.elm-headline {
-		font-weight: 400;
-		font-size: $rea-main-h4-font-size; // * TODO: possibly rework variable naming
+		font-family: $db-font-family-headline;
+		font-weight: 900;
 	}
 }
 h1 {
 	&.elm-headline {
-		font-size: $rea-main-h1-font-size; // * TODO: possibly rework variable naming
+		font-size: to-rem(
+			$pxValue: $rea-main-h1-font-size
+		); // * TODO: possibly rework variable naming
+		line-height: math.div(40, $rea-main-h1-font-size);
+		@media screen and (max-width: 767px) {
+			font-size: to-rem(
+				$pxValue: $rea-main-h1-font-size-small
+			); // * TODO: possibly rework variable naming
+			line-height: 1.14;
+		}
 	}
 }
 h2 {
 	&.elm-headline {
-		font-size: $rea-main-h2-font-size; // * TODO: possibly rework variable naming
+		font-size: to-rem(
+			$pxValue: $rea-main-h2-font-size
+		); // * TODO: possibly rework variable naming
+		line-height: math.div(28, $rea-main-h2-font-size);
+		@media screen and (max-width: 767px) {
+			font-size: to-rem(
+				$pxValue: $rea-main-h2-font-size-small
+			); // * TODO: possibly rework variable naming
+			line-height: 1.2;
+		}
 	}
 }
 h3 {
 	&.elm-headline {
-		font-size: $rea-main-h3-font-size; // * TODO: possibly rework variable naming
+		font-size: to-rem(
+			$pxValue: $rea-main-h3-font-size
+		); // * TODO: possibly rework variable naming
+		line-height: math.div(24, $rea-main-h3-font-size);
+		@media screen and (max-width: 767px) {
+			font-size: to-rem(
+				$pxValue: $rea-main-h3-font-size-small
+			); // * TODO: possibly rework variable naming
+			line-height: 1.33;
+		}
+	}
+}
+h4 {
+	&.elm-headline {
+		font-weight: 400;
+		font-size: to-rem(
+			$pxValue: $rea-main-h4-font-size
+		); // * TODO: possibly rework variable naming
+		line-height: math.div(24, $rea-main-h4-font-size);
+		@media screen and (max-width: 767px) {
+			font-size: to-rem(
+				$pxValue: $rea-main-h4-font-size-small
+			); // * TODO: possibly rework variable naming
+			line-height: 1.25;
+		}
 	}
```

Comment:
should we add default definitions for h5 and h6 as well as mentioned in this issue: https://github.com/db-ui/core/issues/76. In v2 we do not have definitions by design for these but wouldn't it be more comfortable for users to have default settings for those as well? (e.g. as defined in elements: https://db-ui.github.io/elements/?path=/story/04-data-display-headline-intro--page)

Reply:
@annsch but even though lets keep the work out of these two tasks seperate, and implement the definitions for that other issue in a seperate PR (mainly to have this resolved and merged).